### PR TITLE
Remove randomize RBF from wallet UI advanced settings

### PR DESCRIPTION
**Pull request by TChukwuleta**
#5580 

This PR remove from the UI option to randomize RBF

### CODE REVIEW

**Comment by NicolasDorier** [comment]
I think those two properties from the viewmodel aren't used anymore. If so can you just remove them?
```csharp
 vm.SupportRBF = network.SupportRBF;
vm.AllowFeeBump = network.SupportRBF ? WalletSendModel.ThreeStateBool.Yes : WalletSendModel.ThreeStateBool.No;
```

**Comment by TChukwuleta** [comment]
Hello @NicolasDorier 

I have a question: 
Is the goal also to set the RBF flag for PSBT request to true by default, because I see a value assignment on RBF based on if fee bump is enabled in the view model

```
if (network.SupportRBF)
{
    if (sendModel.AllowFeeBump is WalletSendModel.ThreeStateBool.Yes)
        psbtRequest.RBF = true;
    if (sendModel.AllowFeeBump is WalletSendModel.ThreeStateBool.No)
        psbtRequest.RBF = false;
}
```

**Comment by Kukks** [comment]
@NicolasDorier  https://github.com/btcpayserver/btcpayserver/issues/5580#issuecomment-1910560273

**Comment by NicolasDorier** [comment]
> Hello @NicolasDorier
> 
> I have a question: Is the goal also to set the RBF flag for PSBT request to true by default, because I see a value assignment on RBF based on if fee bump is enabled in the view model
> 
> ```
> if (network.SupportRBF)
> {
>     if (sendModel.AllowFeeBump is WalletSendModel.ThreeStateBool.Yes)
>         psbtRequest.RBF = true;
>     if (sendModel.AllowFeeBump is WalletSendModel.ThreeStateBool.No)
>         psbtRequest.RBF = false;
> }
> ```

yes, this should be only `psbtRequest.RBF = true`

**Comment by TChukwuleta** [comment]
Hello @NicolasDorier  

I have removed the properties and refactored

File: BTCPayServer/Controllers/UIWalletsController.PSBT.cs [hunk 1]
```diff
@@ -42,15 +42,7 @@ public async Task<CreatePSBTResponse> CreatePSBT(BTCPayNetwork network, Derivati
...
-
-            if (network.SupportRBF)
-            {
-                if (sendModel.AllowFeeBump is WalletSendModel.ThreeStateBool.Yes)
-                    psbtRequest.RBF = true;
-                if (sendModel.AllowFeeBump is WalletSendModel.ThreeStateBool.No)
-                    psbtRequest.RBF = false;
-            }
-
+            psbtRequest.RBF = true;
```
Comment (NicolasDorier) [inline BTCPayServer/Controllers/UIWalletsController.PSBT.cs:45]: actually you need to keep the check `if (network.SupportRBF)`

Comment (NicolasDorier) [reply BTCPayServer/Controllers/UIWalletsController.PSBT.cs:45]: `psbtRequest.RBF = network.SupportRBF ? true : null;`

Comment (TChukwuleta) [reply BTCPayServer/Controllers/UIWalletsController.PSBT.cs]: Updated

**Pull request merged**